sdk/webhooks/ActionsOnGoogleFulfillment/reading/mockTextData.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Is this just for testing purposes right now, or will all the books be locally stored as a json?

Reply:
I plan on storing all the books locally, originally I was going to host it on firebase's realtime database but that option didn't seem useful vs storing locally.